h/views/api/bulk/_ndjson.py

```diff
@@ -14,6 +14,13 @@ def get_ndjson_response(results: Optional[Iterable]) -> Response:
     if results is None:
         return Response(status=204)
 
+    if not stream:
```

Comment:
Without this the responses are different lines that are json themselves but the whole response is not valid json.

I find that format tricky to consume. We'll keep it in other two endpoints that use it for the performance benefits (which I'm not sure they are that big).

Reply:
The function's comment states its purpose is to return a streaming response for NDJSON payloads.

I think it would make more sense to not add the `stream` flag param here, and instead create a separate helper function that doesn't stream.

Reply:
I think you are right the stream parameters didn't make a lot of sense.

At the end I went just for inlining what I needed in the view. The only thing ndjson about `get_ndjson_response` was setting the right content type :shrug: